fail2ban/tests/files/logs/phpmyadmin-syslog

```diff
@@ -0,0 +1,2 @@
+# failJSON: { "time": "2017-08-22T14:50:22", "match": true , "host": "81.62.21.201" }
```

Comment:
BTW. Usage of real IPs is is considered bad. Suggestion: 192.0.2.1 from test-net-1.